js/main.js

```diff
@@ -117,3 +137,26 @@ function renderSearch(resultsToDisplay) {
     searchResultsContainer.appendChild(searchResultsList);
   });
 }
+
+function renderSearchEmpty(resultsToDisplay) {
+  if (!resultsToDisplay) {
+    const targetElement = document.querySelector('.tag-line');
+    const closestParent = targetElement.closest('.row');
+
+    const noBooksContainer = document.createElement('div');
+    noBooksContainer.classList.add('row');
+
+    const noBooksErrorMessage = document.createElement('p');
+    noBooksErrorMessage.classList.add('no-books-error-message');
+    noBooksErrorMessage.textContent = 'No Books Found! Search Again.';
+
+    closestParent.insertAdjacentElement('afterend',noBooksContainer);
+    noBooksContainer.appendChild(noBooksErrorMessage);
```

Comment:
hmm, this seems like a lot of extra code to display an empty "No Books Found!" Message